website/docs/d/remote_state.html.md

```diff
@@ -31,7 +31,7 @@ resource "aws_instance" "foo" {
 The following arguments are supported:
 
 * `backend` - (Required) The remote backend to use.
-* `environment` - (Optional) The Terraform environment to use.
+* `workspace` - (Optional) The Terraform workspace to use.
```

Comment:
Wondering if it's worth specifying `The workspace has to be explicitly passed in order for it to be possible to access a different remote workspace name.` as per @jbardin 's [comment ](https://github.com/hashicorp/terraform/issues/17153#issuecomment-359592273) 🤔